requirements/dev.txt

```diff
@@ -30,3 +30,5 @@ watchdog==0.9.0 \
 Werkzeug==0.16.0 \
     --hash=sha256:7280924747b5733b246fe23972186c6b348f9ae29724135a6dfc1e53cea433e7 \
     --hash=sha256:e5f4a1f98b52b18a93da705a7458e55afb26f32bff83ff5d19189f92462d65c4
+appnope==0.1.0 \
+   --hash sha256:5b26757dc6f79a3b7dc9fab95359328d5747fcb2409d331ea66d0272b90ab2a0 
```

Comment:
Wasn't sure if you actually wanted to add this to the dependencies... this is what I had to add manually to get the dependencies installed on my machine..